### ISSUE TITLE
How to utilize temperature scaling in binary classification problem with Neural Nets?

### ISSUE THREAD

**Issue by ydzhang12345** [gpleiss/temperature_scaling#8]
Any hints? Or should I just use the traditional method like Platt

**Comment by gpleiss** [comment]
@ydzhang12345 apologies for not responding to this issue earlier! It must've slipped in my notifications.

You can apply temperature scaling as is to binary classification problems. In this case, it is almost exactly identical to Platt scaling. Just apply the temperature to the final output value of the network.

**Comment by ydzhang12345** [comment]
Thank you. I did try that. Turns out using Platt will be better. Thank you!

**Comment by SophieChang66** [comment]
@ydzhang12345 how did you utilize the temperature scaling ? sigmoid(logits / T) or softmax(logits / T) ? what op does your original network to use to compte prob? sigmoid or softmax ?
